codespell_lib/_codespell.py

```diff
@@ -231,14 +255,14 @@ def parse_options(args):
                              'corrections. If this flag is not specified or '
                              'equals "-" then the default dictionary is used. '
                              'This option can be specified multiple times.')
-    builtin_opts = ', '.join(
-        '%r %s' % (d[0], d[1]) for d in _builtin_dictionaries)
+    builtin_opts = '\n- '.join([] + [
```

Comment:
Why do we need the empty array too?

Reply:
To add the prefix for the first line (not having it elsewhere is causing the bug you point out below), otherwise you end up with N-1 `\n- `'s and N entries

Reply:
Ah yes, makes sense, and probably neater than hardcoding it into the string elsewhere.